engine-types/src/types/wei.rs

```diff
@@ -189,6 +189,12 @@ impl From<NEP141Wei> for Wei {
     }
 }
 
+impl From<u128> for Wei {
```

Comment:
I personally don't like adding convenient conversion functions to a newtype like `Wei`. The reason is because I think we should force developers to think about what their `u128` value represents (is it really Wei? Or maybe it is actually ETH or GWei, etc). If we allow a simple `.into()` anywhere we have a `u128` I worry it will not trigger the same thought process as being forced to write `Wei::new(...)`.